rebasehelper/cli.py

```diff
@@ -193,15 +195,17 @@ def build_parser():
             help=argparse.SUPPRESS
         )
```

Comment:
I think we can safery remove `--fedpkg-build-tasks` now. It had been used with upstream monitoring and I don't think anyone else has ever used it.